app/assets/stylesheets/base/layout.css.scss

```diff
@@ -27,6 +27,10 @@ html { overflow-y: auto; }
   &:hover { i { color: white; }}
 }
 
+img, iframe {
+  max-width: 100%;
+}
+
```

Comment:
This won't fuck anything else up will it? I can't imagine it would but side effects are real